### ISSUE TITLE
Disable personal collection

### ISSUE THREAD

**Issue by paoliniluis** [metabase/metabase#18901]
**Is your feature request related to a problem? Please describe.**
A trialer requested if they could disable entirely the "personal collection" functionality, which might make sense in certain situations where you want users who would just consume pre-defined reports

**Describe the solution you'd like**
A toggle to disable personal collections

**Describe alternatives you've considered**
You can make an automated cleanup tool that wipes everything in personal collections, or you can block the API calls as well

**How important is this feature to you?**
Asked by a user who is trialing Metabase

**Additional context**
NA

**Comment by paoliniluis** [comment]
Adding some feedback about this one: user needs to disable personal collections so all users can save questions in a single place only so they can get those questions and show them in a menu

**Comment by brunobergher** [comment]
Thanks helpful context, thank you.

**Comment by brunobergher** [comment]
Never meant to close this, sorry!

**Comment by chaeron** [comment]
From our perspective, providing Metabase dashboards as packaged solutions to clients, we would like to be able to disable personal collections and prevent users from being able to save any questions/dashboards whatsoever.

**Comment by brunobergher** [comment]
@chaeron So your customers would use them in full read-only mode, with no additional exploration, tweaking of filters, time ranges, etc? Or only do those ad hoc without saving their changes, perhaps?

**Comment by chaeron** [comment]
That is correct.  There is no way to remove/disable the personal collection and there is no way to turn off editing of a question by a user using permissions.

**Comment by likeshumidity** [comment]
Spoke with a customer who has a use case for this where they want their users to have "read-only" access to Metabase, so part of that would be disabling the personal collection, but that would also include hiding the "Duplicate" button. Just adding that as an FYI as Duplicate would be useless if a user had no personal collection and also view-only access to all other collections.
